djangocms_versioning/helpers.py

```diff
@@ -206,3 +208,21 @@ def get_editable_url(content_obj):
             model=content_obj._meta.model_name,
         ), args=(content_obj.pk,))
     return url
+
+
+def is_latest_version(content_obj):
+    """Returns True if the object is the latest version in its group, False otherwise.
+    If content_obj is not versioned it returns False
+    """
+    versioning_extension = apps.get_app_config('djangocms_versioning').cms_extension
+    if versioning_extension.is_content_model_versioned(content_obj.__class__):
+        versionable = for_content(content_obj)
+        grouping_values = versionable.grouping_values(content_obj)
+        latest_version = Version.objects.filter_by_grouping_fields(
+            versionable, **grouping_values
+        ).order_by('-pk').first()
+
+        version = Version.objects.get_for_content(content_obj)
+        return latest_version.number == version.number
+    else:
+        return False
```

Comment:
I'd prefer to return early. See this post: https://www.reddit.com/r/programming/comments/878zi2/avoid_else_return_early/